src/pages/Home.jsx

```diff
@@ -1,91 +1,141 @@
+                  <div className='article-meta'>
+                    <a>
+                      <img src={article.author.image} />
+                    </a>
+                    <div className='info'>
+                      <a className='author'>{article.author.username}</a>
+                      <span className='date'>{new Date(article.createdAt).toLocaleDateString()}</span>
+                    </div>
+                    <button
+                      type='button'
+                      className={classNames('btn btn-sm ',{'btn-outline-primary':!article.favorited, 'btn-primary':article.favorited})} // Change to btn-primary if favorited
+                      disabled={false}
+                    >
+                      <i className='ion-heart' />
+                      &nbsp; {article.favoritesCount}
+                    </button>
+                  </div>
+                  <a className='preview-link'>
+                    <h1>{article.title}</h1>
+                    <p>{article.body}</p>
+                    <span>Read more...</span>
+                    <ul className='tag-list'>
+                      {article.tagList.map((tag, idx) => {
+                        return <li key={idx} className='tag-default tag-pill tag-outline'>{tag}</li>
+                      })}
+
+                    </ul>
+                  </a>
                 </div>
-                <button
-                  type="button"
-                  className="btn btn-sm btn-outline-primary" // Change to btn-primary if favorited
-                  disabled={false}
-                >
-                  <i className="ion-heart" />
-                  &nbsp; {faker.datatype.number()}
-                </button>
-              </div>
-              <a className="preview-link">
-                <h1>{faker.lorem.sentence()}</h1>
-                <p>{faker.lorem.paragraph()}</p>
-                <span>Read more...</span>
-                <ul className="tag-list">
-                  <li className="tag-default tag-pill tag-outline">{faker.lorem.word()}</li>
-                </ul>
-              </a>
-            </div>
+              </React.Fragment>
+            })}
+
             <nav>
-              <ul className="pagination">
-                <li className="page-item">
-                  <button type="button" className="page-link">
-                    1
-                  </button>
-                </li>
-                <li className="page-item">
-                  <button type="button" className="page-link">
-                    2
-                  </button>
-                </li>
-                <li className="page-item active">
-                  <button type="button" className="page-link">
-                    3
-                  </button>
-                </li>
+
+              <ul className='pagination'>
+                {Array(pages).fill(0).map((_, i) => i + 1)
+                  .map((page) => {
+                    return <li onClick={() => {
```

Comment:
I would probably inline this... You know, use () instead of {}, inside JSX it's a little easier to read IMO.